Examples/Swift/OrnamentsLayoutExample.swift

```diff
@@ -0,0 +1,30 @@
+import Mapbox
+
+@objc(OrnamentsLayoutExample_Swift)
+class OrnamentsLayoutExample_Swift: UIViewController, MGLMapViewDelegate {
+    private var mapView: MGLMapView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Initializing map view
```

Comment:
It's nice to say where the map will be centered in this comment, since users may not know where the coordinates are for.

Reply:
Sure, will update!